improver/cli/nbhood.py

```diff
@@ -69,7 +69,7 @@ def process(
             neighbourhood is calculated. If "percentiles" is selected, then
             the percentiles are calculated with a neighbourhood. Calculating
             percentiles from a neighbourhood is only supported for a circular
-            neighbourhood.
+            neighbourhood, and the input cube should be ensemble realizations.
```

Comment:
Perhaps, following @NMC-DAVE's suggestion, we could add a sentence, something like "The calculation of percentiles from a neighbourhood is notably slower than neighbourhood processing using a thresholded probability field." ? This would give a user an idea about the expected behaviour.

Reply:
Added